src/components/uploader/OperationUploader.tsx

```diff
@@ -110,14 +111,29 @@ export const OperationUploader: ComponentType = withSuspensable(() => {
         return
       }
 
-      // bypass jsonschema validation according to mist
-      // const jsonSchemaValidation = copilotSchemaValidator.validate(
-      //   'copilot',
-      //   operationObject,
-      // )
-      // console.log(jsonSchemaValidation, copilotSchemaValidator.errors)
       const patchedOperation = operationPatch(operationObject, levels)
       console.log('patchedOperation', patchedOperation)
+
+      const jsonSchemaValidation = copilotSchemaValidator.validate(
+        'copilot',
+        operationObject,
+      )
+      console.log(
+        'jsonSchemaValidationResult',
+        jsonSchemaValidation,
+        'errors',
+        copilotSchemaValidator.errors,
+      )
+
+      if (!jsonSchemaValidation && copilotSchemaValidator.errors) {
+        setErrors(
+          copilotSchemaValidator.errors.map(
+            (error) =>
+              `Path: ${error.instancePath}, Error: ${error.message}` ?? '',
```

Comment:
Consider use more descriptive languages to inform the user about the location of errors.

Reply:
use ajv-i18n